showcases/ui5-tsapp/webapp/test/integration/HelloJourney.ts

```diff
@@ -0,0 +1,48 @@
+import opaTest from "sap/ui/test/opaQunit";
+import MainPage from "./pages/Main";
+
+const onTheMainPage = new MainPage();
+
+QUnit.module("Hello");
+
+opaTest("Should open the Hello dialog", function () {
+	// Arrangements
+	onTheMainPage.iStartMyUIComponent({
+		componentConfig: {
+			name: "ui5.ecosystem.demo.tsapp",
+		},
+	});
+
+	// Actions
+	onTheMainPage.iPressTheSayHelloButton();
+
+	// Assertions
+	onTheMainPage.iShouldSeeTheHelloDialog();
+
+	// Actions
+	onTheMainPage.iPressTheOkButtonInTheDialog();
+
+	// Assertions
+	onTheMainPage.iShouldNotSeeTheHelloDialog();
+
+	// Cleanup
+	onTheMainPage.iTeardownMyApp();
+});
+
+opaTest("Should close the Hello dialog", function (Given, When, Then) {
```

Comment:
ummm... didn't we get rid of Given/When/Then? It's also not used.

Reply:
Seems I missed to remove it at this place... THX for the catch